### PR TITLE
[FEATURE] Order wins and losses in Stats page

### PR DESCRIPTION
**Pull request by amina-alibasic**

### CODE REVIEW

**Comment by amina-alibasic** [comment]
@itsalaidbacklife let me know if tests need to be updated as well. Current test case:
```
 cy.get('[data-players-lost-to=Player3-week-total]').should(
      'contain',
      'Player1 (2), Player2 (2), Player4 (2), Player5 (1)'
    );
```
seems to be fine for the sorting of losses. And for the wins, there's just 1 win for this player in this test case.

**Review by itsalaidbacklife — CHANGES_REQUESTED** [review]
Looks right. Can you update a test case to check the sort order on the menu when clicking the total column?

File: tests/e2e/fixtures/statsFixtures.js [hunk 1]
```diff
@@ -94,6 +94,13 @@ const seasonOneMatches = [
     startTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
     endTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
   },
+  {
+    player1: 'player1',
+    player2: 'player3',
+    winner: 'player1',
+    startTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2022-01-20', 'yyyy-mm-dd').valueOf(),
+  },
```
Comment (itsalaidbacklife) [inline tests/e2e/fixtures/statsFixtures.js:103]: Can you move this up above with the other player 1 week 1 matches?

**Review by itsalaidbacklife — APPROVED** [review]
Looks great. Just requesting the one new match in the fixture get moved above to where the other player 1 matches are

File: tests/e2e/fixtures/statsFixtures.js (hunk 1 above)
Comment (amina-alibasic) [reply tests/e2e/fixtures/statsFixtures.js:103]: Done

**Pull request merged**